### PR TITLE
DDF-4121 Makes query result item extensible

### PR DESCRIPTION
**Pull request by djblue**
#### What does this PR do?

Makes query result item extensible

#### Who is reviewing it?

@gjvera 
@GabrielFabian 
@millerw8 
@willwill96 

#### Select relevant component teams: 

@codice/ui 

#### Ask 2 committers to review/merge the PR and tag them here.

@andrewkfiedler
@bdeining

#### How should this be tested?

- `mvn clean install` ui directory
- ingest data with stereo pair
- replicate demo

#### What are the relevant tickets?

[DDF-4121](https://codice.atlassian.net/browse/DDF-4121)

#### Screenshots

[demo](https://youtu.be/Jv2TeIyJyQE)

#### Checklist:

- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/stereo-pairs/index.tsx [hunk 1]
```diff
@@ -0,0 +1 @@
+export * from './stereo-pairs'
```
Comment (willwill96) [inline ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/stereo-pairs/index.tsx]: Should this have a copyright in it?

File: ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/stereo-pairs/stereo-pairs.tsx [hunk 2]
```diff
@@ -0,0 +1,274 @@
...
+
+type PreviewProps = {
+  pair: {
+    thumbnail: string
+    title: string
+  }
+}
+
+const Preview = ({ pair }: PreviewProps) =>
+  pair === null ? (
```
[the remark this answers is not in the document]
Comment (djblue) [reply ui/packages/catalog-ui-search/src/main/webapp/react-component/presentation/stereo-pairs/stereo-pairs.tsx]: Good catch!

**Comment by GabrielFabian** [comment]
build now

**Comment by djblue** [comment]
build now

**Pull request merged**